.travis.yml

```diff
@@ -10,6 +10,8 @@ matrix:
       env: ARCH=ppc64le
     - name: "ARCH=x86_64"
       env: ARCH=x86_64
+    - name: "ARCH=x86_64 LD=ld.lld"
+      env: ARCH=x86_64 LD=ld.lld-8
```

Comment:
Once these are green on presubmit (and you can point to a build number that shows they're green), let's demote `ARCH=x86_64` and `ARCH=x86_64 LD=ld.lld REPO=linux-next` to cron (keeping LLD in presubmit).